checker/jdk/Makefile.jdk

```diff
@@ -62,7 +62,7 @@ validate: classes $(DIFFS)
 # does not exist.  So long as the variables are not used on such a make
 # invocation, this doesn't cause a problem, but it is irritating (and a bit
 # sloppy).
-BUILD_CLASSES = $(shell find build/java build/javax -name '*.class')
+BUILD_CLASSES = $(shell find build/java build/javax build/org/w3c build/org/xml -name '*.class')
```

Comment:
Depending on whether there are any non-JDK classes in these directories, it might be easier just to use the top-level directory (e.g. `src` or `build`) instead? I'm not too familiar with `make`, so I'm not seeing an obvious way to implement this. Maybe something with sub-shells or `env` will work?